src/cluster.c

```diff
@@ -1626,3 +1626,144 @@ void clusterCommandFlushslot(client *c) {
     delKeysInSlot(slot, lazy, false, true);
     addReply(c, shared.ok);
 }
+
+/* -----------------------------------------------------------------------------
+ * CLUSTERSCAN Command
+ * -------------------------------------------------------------------------- */
+ static const char *getHashTagForSlot(int slot) {
```

Comment:
```suggestion
 static const char *getHashTagForSlot(int slot) {
```

I think you can use `return (slot < CLUSTER_SLOTS) ? crc16_slot_table[slot] : NULL;` instead. You can see the implementation of VM_ClusterCanonicalKeyNameInSlot as well